release/models/platform/openconfig-platform-healthz.yang

```diff
@@ -38,6 +38,12 @@ module openconfig-platform-healthz {
     reference "0.1.0";
   }
 
+  revision "2023-04-11" {
+    description
+      "Clarification for healthz state transition and unhealthy-count leaf";
+    reference "0.1.1";
+  }
+
```

Comment:
Hi,  (lint check caught these issues:

* You must place this text above the previous revision statement (ie: move this text to line 35).  
* You also must update line 33 to match.

Reply:
sorry, @dplore 
Fixed